### ISSUE TITLE
Failed to run `make bundle with v1.18.0

### ISSUE THREAD

**Issue by Madhu-1** [operator-framework/operator-sdk#5574]
#### What did you do?

Running `make bundle` on the project with the latest operator SDK is failing

#### What did you expect to see?

After updating the operator SDK version the existing `make bundle` should not be breaking

* We have update the labels on the all the yaml files in our project
* Update operator-sdk from 1.17.0 to 1.18.0 and `make bundle` is failing 
```
[🎩︎]mrajanna@fedora kubernetes-csi-addons $]make bundle
sed 's/@PACKAGE_NAME@/csi-addons/g' < config/manifests/bases/clusterserviceversion.yaml.in > config/manifests/bases/csi-addons.clusterserviceversion.yaml
go build -o /home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/controller-gen ./vendor/sigs.k8s.io/controller-tools/cmd/controller-gen
go build -o /home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/kustomize ./vendor/sigs.k8s.io/kustomize/kustomize/v4
/home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/controller-gen rbac:roleName=manager-role crd webhook paths="{./api/...,./cmd/...,./controllers/...,./sidecar/...}" output:crd:artifacts:config=config/crd/bases
cd config/manager && /home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/kustomize edit set image controller=quay.io/csiaddons/k8s-controller:latest rbac-proxy=gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
/home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/kustomize build config/crd > deploy/controller/crds.yaml
/home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/kustomize build config/rbac > deploy/controller/rbac.yaml
/home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/kustomize build config/manager > deploy/controller/setup-controller.yaml
go build -o /home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/operator-sdk ./vendor/github.com/operator-framework/operator-sdk/cmd/operator-sdk
/home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/kustomize build config/manifests | /home/mrajanna/workspace/src/github.com/csi-addons/kubernetes-csi-addons/bin/operator-sdk generate bundle --manifests --metadata --package=csi-addons 
Generating bundle manifests
FATA[0000] Error generating bundle manifests: could not find manager deployment, should have label control-plane=controller-manager 
make: *** [Makefile:96: bundle] Error 1
```
#### Environment

**Operator type:** go

**Kubernetes cluster type:**

`$ operator-sdk version` v1.18.0

`$ go version` (if language is Go)  go1.17.4

`$ kubectl version`

**Comment by nixpanic** [comment]
It seems that the change was introduced with #5567. The labels to detect the `manager` container are hard-codedL

https://github.com/operator-framework/operator-sdk/blob/c9c61b6921b29d731e64cd3cc33d268215fb3b25/internal/cmd/operator-sdk/generate/bundle/bundle.go#L322-L328

In csi-addons/kubernetes-csi-addons#122 we replaced those labels with our own, as to not cause confusion or conflicts with components that use the standard labels.This might not have been the best approach, but it seemed the most practical.

A recommendation or suggested workaround that makes sense from an operator-framework point of view would be much appreciated.

**Comment by camilamacedo86** [comment]
@ryantking, 

why do we need to use the labels to get the manifest? why have we are not looking at the path since the manager will be always in the path [config](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/config)/[manager](https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/config/manager)/manager.yaml?

**Comment by camilamacedo86** [comment]
Hi @ryantking,

Following some questions, in order to be able to check this one. I hope that can help out. 

Why do we need to use the labels to get the manifests? The logic to ensure that all will be mapped via RELATED_IMAGE should not be valid for all images used? Are we not doing the same for the [kube-rbac-proxy](https://github.com/operator-framework/operator-sdk/blob/master/testdata/go/v3/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml#L115-L133) If not, why not?

Also, wdyt about creating a sample under testdata dir to scaffold the bundle using. this option for we are able to visualize to check its results and be able to easier verify in the review that changes will not affect this feature?  

c/c @jmrodri

**Comment by acornett21** [comment]
I'm interested in a resolution for this as well, as this is a breaking change for all ACK operators.

**Comment by pohly** [comment]
Hardcoding these labels in operator-sdk seems like a limitation that should be addressed, in particular because the defaults don't follow the [recommended naming scheme](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/).

But that aside, which objects should have this label? Looking at the examples under testdata it seems that all (?) have it?

**Comment by ryantking** [comment]
This is fixed in [v1.19.1](https://github.com/operator-framework/operator-sdk/releases/tag/v1.19.1).
